### PR TITLE
docs: add space between sentences

### PR DESCRIPTION
**Pull request by mortezahatamikia-code**
just added a space before "Here's a list of..." for a better reading experience

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/1.docs/1.getting-started/6.data-fetching.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ Be very careful before proxying headers to an external API and just include head
 
 Not all headers are safe to be bypassed and might introduce unwanted behavior.
```
Comment (danielroe) [inline docs/content/1.docs/1.getting-started/6.data-fetching.md:249]:
```suggestion
Not all headers are safe to be bypassed and might introduce unwanted behavior. Here is a list of common headers that are NOT to be proxied:
```

File: docs/content/1.docs/1.getting-started/6.data-fetching.md [hunk 2]
```diff
@@ -248,7 +248,7 @@ Be very careful before proxying headers to an external API and just include head
...
 
-Here is a list of common headers that are NOT to be proxied:
+ Here is a list of common headers that are NOT to be proxied:
```
Comment (danielroe) [inline docs/content/1.docs/1.getting-started/6.data-fetching.md:252]:
```suggestion
```

**Review by danielroe — COMMENTED** [review]
thanks!

**Review by danielroe — APPROVED** [review]

**Review by Atinux — APPROVED** [review]

**Comment by mortezahatamikia-code** [comment]
> thanks!

your welcome!

**Pull request merged**